### ISSUE TITLE
Support for _include in compartments

### ISSUE THREAD

**Issue by orlandow** [Aidbox/Issues#24]
Including practitioner in compartment query:

```
┌────
│ GET /Organization/org2/PractitionerRole?_include=PractitionerRole:practitioner
└────

┌────
│ {
│   "organization": {
│     "id": "org2",
│     "resourceType": "Organization"
│   },
│   "practitioner": {
│     "id": "pract2",
│     "resourceType": "Practitioner"
│   },
│   "id": "role3",
│   "resourceType": "PractitionerRole",
│   "meta": {
│     "lastUpdated": "2018-11-26T17:17:27.472Z",
│     "versionId": "553",
│     "tag": [
│       {
│         "system": "https://aidbox.app",
│         "code": "created"
│       }
│     ]
│   }
│ }
└────
```

Regular query:

```
┌────
│ GET /PractitionerRole?organization=org2&_include=PractitionerRole:practitioner
└────

┌────
│ [
│   {
│     "resource": {
│       "organization": {
│         "id": "org2",
│         "resourceType": "Organization"
│       },
│       "practitioner": {
│         "id": "pract2",
│         "resourceType": "Practitioner"
│       },
│       "id": "role3",
│       "resourceType": "PractitionerRole",
│       "meta": {
│         "lastUpdated": "2018-11-26T17:17:27.472Z",
│         "versionId": "553",
│         "tag": [
│           {
│             "system": "https://aidbox.app",
│             "code": "created"
│           }
│         ]
│       }
│     },
│     "fullUrl": "/PractitionerRole/role3"
│   },
│   {
│     "resource": {
│       "id": "pract2",
│       "resourceType": "Practitioner",
│       "meta": {
│         "lastUpdated": "2018-11-26T17:13:35.657Z",
│         "versionId": "550",
│         "tag": [
│           {
│             "system": "https://aidbox.app",
│             "code": "updated"
│           }
│         ]
│       }
│     },
│     "fullUrl": "/Practitioner/pract2"
│   }
│ ]
└────
```

**Comment by orlandow** [comment]
## Example using patient compartment definition

```
┌────
│ GET /Patient/pat2/Observation?_include=Observation:performer
└────

┌────
│ obs5
└────

-------------------------------------------

┌────
│ GET /Observation?subject=pat2&_include=Observation:performer
└────

┌────
│ obs5
│ pract2
└────
```

Note: these queries are extracting resource ids (` | jq .entry[].resource.id`)

**Comment by mlapshin** [comment]
fixed in version `0.3.3-120-g342df9e`
